changelog/README.rst

```diff
@@ -0,0 +1,35 @@
+:orphan:
+
+Changelog
+=========
+
+This directory contains "news fragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next what's new page.
+
+Make sure to use full sentences with correct case and punctuation, and please
+try to use Sphinx intersphinx using backticks.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``api_chage``: A change which requires users to change code and is not
+  backwards compatible. (Not to be used for removal of deprecated features.)
+* ``feature``: New user facing features and any new behavior.
+* ``removal``: Removal of a deprecated part of the API.
```

Comment:
It may be worth adding a `misc` category here. I find that a pretty significant fraction of my `changelog.d` files ends up being something like "changed `tox.ini`" or other miscellaneous stuff that isn't user-facing at all.

I currently don't actually have those `misc` entries go into the final changelog, but I'm considering putting them in under a `<details>` tag or something.